packages/contracts-bedrock/test/L1/OPContractsManagerStandardValidator.t.sol

```diff
@@ -236,34 +236,88 @@ abstract contract OPContractsManagerStandardValidator_TestInit is CommonTest, Di
                 address owner = proxyAdmin.owner();
 
                 // Prepare the upgrade input.
-                IOPContractsManagerV2.DisputeGameConfig[] memory disputeGameConfigs =
-                    new IOPContractsManagerV2.DisputeGameConfig[](3);
-                disputeGameConfigs[0] = IOPContractsManagerV2.DisputeGameConfig({
-                    enabled: true,
-                    initBond: disputeGameFactory.initBonds(GameTypes.CANNON),
-                    gameType: GameTypes.CANNON,
-                    gameArgs: abi.encode(IOPContractsManagerV2.FaultDisputeGameConfig({ absolutePrestate: cannonPrestate }))
-                });
-                disputeGameConfigs[1] = IOPContractsManagerV2.DisputeGameConfig({
-                    enabled: true,
-                    initBond: disputeGameFactory.initBonds(GameTypes.PERMISSIONED_CANNON),
-                    gameType: GameTypes.PERMISSIONED_CANNON,
-                    gameArgs: abi.encode(
-                        IOPContractsManagerV2.PermissionedDisputeGameConfig({
-                            absolutePrestate: cannonPrestate,
-                            proposer: proposer,
-                            challenger: challenger
-                        })
-                    )
-                });
-                disputeGameConfigs[2] = IOPContractsManagerV2.DisputeGameConfig({
-                    enabled: true,
-                    initBond: disputeGameFactory.initBonds(GameTypes.CANNON_KONA),
-                    gameType: GameTypes.CANNON_KONA,
-                    gameArgs: abi.encode(
-                        IOPContractsManagerV2.FaultDisputeGameConfig({ absolutePrestate: cannonKonaPrestate })
-                    )
-                });
+                IOPContractsManagerV2.DisputeGameConfig[] memory disputeGameConfigs = isDevFeatureEnabled(
+                    DevFeatures.OPTIMISM_PORTAL_INTEROP
+                ) ? new IOPContractsManagerV2.DisputeGameConfig[](5) : new IOPContractsManagerV2.DisputeGameConfig[](3);
+                if (isDevFeatureEnabled(DevFeatures.OPTIMISM_PORTAL_INTEROP)) {
+                    disputeGameConfigs[0] = IOPContractsManagerV2.DisputeGameConfig({
```

Comment:
We can reduce the amount of code needed here if the size of the array is dynamic. Would make this easier to maintain for future developers. 

```solidity
new IOPContractsManagerV2.DisputeGameConfig[](isDevFeatureEnabled(DevFeatures.OPTIMISM_PORTAL_INTEROP) ? 5 : 3)
```

Then the IF statement is only used to assign to the last two game types.

Reply:
Good call.
86dfa567c375